diff --git a/src-tauri/gen/android/app/build.gradle.kts b/src-tauri/gen/android/app/build.gradle.kts
index 66f3c7a..15c4118 100644
--- a/src-tauri/gen/android/app/build.gradle.kts
+++ b/src-tauri/gen/android/app/build.gradle.kts
@@ -51,6 +51,16 @@ android {
     buildFeatures {
         buildConfig = true
     }
+
+    dependenciesInfo {
+        includeInApk = false
+        includeInBundle = false
+    }
+    splits {
+        abi {
+            isEnable = false
+        }
+    }
 }
 
 rust {
